egs/librispeech/asr1/run.sh

```diff
@@ -11,6 +11,7 @@ backend=pytorch
 stage=-1       # start from -1 if you need to start from data download
 stop_stage=100
 ngpu=4         # number of gpus ("0" uses cpu, otherwise use gpu)
+nj=32
```

Comment:
I agree. When I test the recipe in my small PC, the configurable `nj` really helps :sob:. However, the other recipes might not have such a high load average problem on this (because LibriSpeech is the biggest dataset in ESPnet.)

Reply:
in tts recipes, `nj` is hyperparameter :)